server/controllers/finance/reports/shared.js

```diff
@@ -0,0 +1,85 @@
+
+  { field : 'is_caution', displayName : 'FORM.LABELS.CAUTION' },
+  { field : 'supplier_uuid', displayName : 'FORM.LABELS.SUPPLIER' },
+  { field : 'status_id', displayName : 'PURCHASES.ORDER' },
+  { field : 'patientReference', displayName : 'FORM.LABELS.REFERENCE_PATIENT' },
+  { field : 'user_id', displayName : 'FORM.LABELS.USER' },
+
+  { field : 'invoiceReference', displayName : 'FORM.LABELS.INVOICE' },
+  { field : 'invoice_uuid', displayName : 'FORM.LABELS.INVOICE' },
+  { field : 'service_id', displayName : 'FORM.LABELS.SERVICE' },
+  { field : 'inventory_uuid', displayName : 'FORM.LABELS.INVENTORY' },
+
+  { field : 'flux_id', displayName: 'STOCK.FLUX' },
+  { field : 'status', displayName: 'STOCK.STATUS.LABEL' },
+  { field : 'hospital_no', displayName : 'FORM.LABELS.HOSPITAL_NO' },
+  { field : 'patient_group_uuid', displayName : 'PATIENT_GROUP.PATIENT_GROUP' },
+
+  { field : 'entry_date_from', displayName: 'STOCK.ENTRY_DATE', comparitor: '>', isDate : true },
+  { field : 'entry_date_to', displayName: 'STOCK.ENTRY_DATE', comparitor: '<', isDate : true },
+  { field : 'expiration_date_from', displayName: 'STOCK.EXPIRATION_DATE', comparitor: '>', isDate : true },
+  { field : 'expiration_date_to', displayName: 'STOCK.EXPIRATION_DATE', comparitor: '<', isDate : true },
+  { field : 'is_exit', displayName : 'STOCK.OUTPUT' },
+  { field : 'label', displayName : 'STOCK.LOT' },
+  { field : 'depot_uuid', displayName : 'STOCK.DEPOT' },
+
+  { field : 'description', displayName : 'FORM.LABELS.DESCRIPTION' },
+  { field : 'entity_uuid', displayName : 'FORM.LABELS.ENTITY' },
+
+  { field : 'display_name', displayName : 'FORM.LABELS.NAME' },
+  { field : 'sex', displayName : 'FORM.LABELS.GENDER' },
+  { field : 'code', displayName : 'FORM.LABELS.CODE' },
+  { field : 'dateBirthFrom', displayName : 'FORM.LABELS.DOB', comparitor : '>', isDate : true },
+  { field : 'dateBirthTo', displayName : 'FORM.LABELS.DOB', comparitor : '<', isDate : true },
+  { field : 'dateEmbaucheFrom', displayName : 'FORM.LABELS.DATE_EMBAUCHE', comparitor : '>', isDate : true },
+  { field : 'dateEmbaucheTo', displayName : 'FORM.LABELS.DATE_EMBAUCHE', comparitor : '<', isDate : true },
+  { field : 'grade_id', displayName : 'FORM.LABELS.GRADE' },
+  { field : 'fonction_id', displayName : 'FORM.LABELS.FUNCTION' },
+  { field : 'service_id', displayName : 'FORM.LABELS.SERVICE' },
+  { field : 'type_ids', displayName : 'FORM.LABELS.TRANSACTION_TYPE' },
+
+  { field : 'reference', displayName : 'FORM.LABELS.REFERENCE' },
+  { field : 'reversed', displayName : 'CASH.REGISTRY.REVERSED_RECORDS' },
+  { field : 'limit', displayName : 'FORM.LABELS.LIMIT' },
+  { field : 'period', displayName : 'TABLE.COLUMNS.PERIOD', isPeriod : true },
+  { field : 'custom_period_start', displayName : 'PERIODS.START', isDate : true, comparitor : '>' },
+  { field : 'custom_period_end', displayName : 'PERIODS.END', isDate : true, comparitor : '<' },
+];
+
+_.sortBy(columns, c => c.field);
```

Comment:
I can not see the benefit of sorting these items, you can remove it and avoid an unnecessary operation.

Reply:
:+1: